assets/dso-toolkit-scripts.js

```diff
@@ -7,3 +11,136 @@ function DsoToolkitResizeIframe(frame, element) {
 
   frame.style.height = (height + 32) + 'px';
 }
+
+(function () {
+  var cleanRegex = /[ -]/g;
+
+  Vue.component('dso-navigation-filter', {
+    template:
+      '<div class="dso-search-bar">' +
+        '<div class="dso-search-bar-input">' +
+          '<span class="dso-search-icon" aria-hidden="true"></span>' +
+          '<input type="text" placeholder="Filter op component" v-model.trim="input">' +
+          '<button type="button" @click="clearFilter()" title="Zoekopdracht legen">' +
+            'Zoekopdracht legen' +
+          '</button>' +
+        '</div>' +
+      '</div>',
+    data: function () {
+      return {
+        input: localStorage.getItem('filterValue') || '',
+        items: Array.prototype.slice.call(document.querySelectorAll('[data-role="item"]')),
```

Comment:
Is het menu niet eenduidiger te selecteren dan met `document.querySelectorAll('[data-role="item"]')` ?

Reply:
Dus, door die items te selecteren onder de huidige `.dso-tree-view` die we gaan 'besturen'?

Reply:
Niet onder de huidige `.dso-tree-view`, ik neem voor nu aan dat er maar één tree is. Kun je jouw eerste reactie verder verduidelijken? De query selector is hetzelfde. Stel je voor om er geen array van te maken?